.storybook/addons.js

```diff
@@ -1,4 +1,2 @@
-import '@storybook/addon-options/register';
 import '@storybook/addon-actions/register';
 import '@storybook/addon-a11y/register';
```

Comment:
It looks like we removed all the places the corresponding decorator is used. Should we strip this out too, or are we planning on adding them back in piecemeal?

Reply:
The new version of this addon doesn't need to be added with addDecorator – it seems this line is all you need to get set up.